meta/runtime.yml

```
@@ -43,3 +43,212 @@ action_groups:
     - icinga_user_template_info
     - icinga_zone
     - icinga_zone_info
+
+plugin_routing:
+  inventory:
+    icinga_director_inventory:
+      redirect: telekom_mms.icinga_director.icinga_director_inventory
+      deprecation:
+        warning_text: Use telekom_mms.icinga_director.icinga_director_inventory instead
+        removal_version: 2.0.0
```

This is the version when the redirect will be removed:
```suggestion
        removal_version: 3.0.0
```
Same below.

---

Thanks, I changed it to 3.0.0 everywhere.